src/Command/CopyAssetsCommand.php

```diff
@@ -77,22 +77,20 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             try {
                 $this->filesystem->remove($targetDir);
                 $this->hardCopy($originDir, $targetDir);
-
+                dump($originDir, $targetDir);
```

Comment:
Ooops, that probably shouldn't be here @bobdenotter ?

Reply:
Thx!